.helm/starter/templates/awx-deploy.yaml

```diff
@@ -5,6 +5,14 @@ kind: AWX
 metadata:
   name: {{ .name }}
   namespace: {{ $.Release.Namespace }}
+  {{- with .labels }}
+  labels:
+    {{ . | toYaml }}
+  {{- end }}
+  {{- with .annotations }}
+  labels:
```

Comment:
Typo: it should be `annotations`.

Reply:
@Jibux you are correct, fixing in https://github.com/ansible-community/awx-operator-helm/pull/28